include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_merge.h

```diff
@@ -133,17 +133,18 @@ __find_start_point(const _Rng1& __rng1, const _Index __rng1_from, _Index __rng1_
 // Do serial merge of the data from rng1 (starting from start1) and rng2 (starting from start2) and writing
 // to rng3 (starting from start3) in 'chunk' steps, but do not exceed the total size of the sequences (n1 and n2)
 template <typename _Rng1, typename _Rng2, typename _Rng3, typename _Index, typename _Compare>
-void
+std::pair<oneapi::dpl::__internal::__difference_t<_Rng1>, oneapi::dpl::__internal::__difference_t<_Rng2>>
```

Comment:
I think it's enough to return from this function `std::pair` of real used typed:
```C++
std::pair<_Index, _Index>
```
without any extra conversions inside.